src/swift_server.rs

```diff
@@ -185,6 +195,108 @@ pub async fn process_order(
                 }),
             );
         }
+    };
+
+    if user.is_none() {
+        log::warn!(
+            target: "server",
+            "{log_prefix}: Error simulating order, user not found"
+        );
+        return (
+            axum::http::StatusCode::BAD_REQUEST,
+            Json(ProcessOrderResponse {
+                message: PROCESS_ORDER_RESPONSE_ERROR_USER_NOT_FOUND,
+                error: Some("user not found".to_string()),
+            }),
+        );
+    }
+    let user = user.unwrap();
+
+    // If fat fingered order that requires sanitization, then just send the order
+    let mut order_params = order_params.clone();
+    if server_params
+        .simulate_will_auction_params_sanitize(&mut order_params)
+        .await
+    {
+        let uuid = String::from_utf8(uuid.to_vec()).expect("invalid utf8 uuid");
+        let tx_builder = TransactionBuilder::new(
+            server_params.drift.program_data(),
+            taker_pubkey,
+            std::borrow::Cow::Owned(user),
+            false,
+        )
+        .with_priority_fee(
+            server_params.priority_fee_subscriber.priority_fee(),
+            Some(1_400_000),
+        );
+
+        let versioned_message = match *signed_msg {
+            SignedMsgType::Authority(signed_order) => {
+                let signed_order_info = SignedOrderInfo::new(
+                    uuid,
+                    taker_authority,
+                    signing_pubkey,
+                    signed_order,
+                    Signature::from(taker_signature),
+                );
+                tx_builder
+                    .place_swift_order(&signed_order_info, &user)
+                    .build()
+            }
+            SignedMsgType::Delegated(signed_order) => {
+                let signed_order_info = SignedDelegateOrderInfo::new(
+                    uuid,
+                    taker_authority,
+                    signing_pubkey,
+                    signed_order,
+                    Signature::from(taker_signature),
+                );
+                tx_builder
+                    .place_swift_delegate_order(&signed_order_info, &user)
+                    .build()
+            }
+        };
+
+        let place_tx_with_timeout = tokio::time::timeout(
+            Duration::from_secs(35),
+            send_tx(
+                &server_params.drift,
+                versioned_message,
+                "place swift order",
+                Some(30),
+                log_prefix.clone(),
+            ),
+        )
+        .await;
```

Comment:
`send_tx` spawns a background task so this timeout is only awaiting the setup/spawn not the actual tx sending